src/relay/backend/aot_executor_codegen.cc

```diff
@@ -48,6 +48,7 @@ namespace backend {
 using IntegerArray = Array<Integer>;
 using StorageMap =
     std::unordered_map<Expr, StorageInfo, runtime::ObjectPtrHash, runtime::ObjectPtrEqual>;
+using namespace tec;
```

Comment:
(stylistic question) It looks like this is an established pattern in the code but the style guide discourages this:
https://google.github.io/styleguide/cppguide.html#Namespaces

Reply:
Sure, I can use tec:: instead